### PR TITLE
local sandbox tmp dir supports windows

### PR DESCRIPTION
**Pull request by cosmicBboy**
Signed-off-by: cosmicBboy <niels.bantilan@gmail.com>

# TL;DR
fixes https://github.com/flyteorg/flyte/issues/782

## Type
 - [X] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description

The default filepath defined in by `LOCAL_SANDBOX` resulted in an usable tmp-dir path when invoking `flytekit.current_context().working_directory`

## Tracking Issue
fixes https://github.com/flyteorg/flyte/issues/782

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

**Review by EngHabu — APPROVED** [review]

File: flytekit/configuration/sdk.py [hunk 1]
```diff
@@ -20,7 +23,11 @@
 TODO: Explain how this would be used to extend the SDK
 """
 
-LOCAL_SANDBOX = _config_common.FlyteStringConfigurationEntry("sdk", "local_sandbox", default="/tmp/flyte")
+LOCAL_SANDBOX = _config_common.FlyteStringConfigurationEntry(
```
Comment (kumare3) [inline flytekit/configuration/sdk.py:26]: can we not use TempDir - https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory

**Review by kumare3 — APPROVED** [review]
